src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,54 @@
-export const Tabs = () => {};
+import React from 'react';
+import { Tab } from '../typeTab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  onTabSelected:Function,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const findTab = tabs.find(tab => tab.id === selectedTabId);
+
+  const tabId = findTab
```

Comment:
you don't need this check here. If you want to ensure that your active tab is always valid, just do it inside your setAnotherTab function

Reply:
I have to check ıt ın Tabs component, this function is a prop only from another component. And according to test it should stay as a prop in another component
![image](https://user-images.githubusercontent.com/96202644/218325731-d323cb35-b6c3-432e-be13-c71f05aca8bf.png)